src/test/java/core/basesyntax/service/impl/CsvDataParserImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.OperationType;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CsvDataParserImplTest {
+    private static final String COLUMN_SEPARATOR = ",";
+    private static final DataParser parser = new CsvDataParserImpl(COLUMN_SEPARATOR);
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @Test
+    public void parse_balanceStringToParse_ok() {
+        String stringToParse = "b,banana,22";
+        FruitTransaction expected = new FruitTransaction(OperationType.BALANCE,
+                "banana", 22);
+        FruitTransaction actual = parser.parse(stringToParse);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parse_returnStringToParse_ok() {
+        String stringToParse = "r,banana,2";
+        FruitTransaction expected = new FruitTransaction(OperationType.RETURN,
+                "banana", 2);
+        FruitTransaction actual = parser.parse(stringToParse);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parse_purchaseStringToParse_ok() {
+        String stringToParse = "p,banana,12";
+        FruitTransaction expected = new FruitTransaction(OperationType.PURCHASE,
+                "banana", 12);
+        FruitTransaction actual = parser.parse(stringToParse);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parse_supplyStringToParse_ok() {
+        String stringToParse = "s,banana,12";
+        FruitTransaction expected = new FruitTransaction(OperationType.SUPPLY,
+                "banana", 12);
+        FruitTransaction actual = parser.parse(stringToParse);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void stringContainsMaxIntegerValue_ok() {
```

Comment:
add the name of the method you test to signature. Like in previous ones.  Also true for remaining methods.
`parse_stringContainsMaxIntegerValue_ok`